src/v2/normalized_packages.py

```diff
@@ -53,3 +57,65 @@ def dependency_graph(self) -> Dict[Package, Set[Package]]:
     def ecosystem(self):
         """Ecosystem value."""
         return self._ecosystem
+
+
+class GoNormalizedPackages(NormalizedPackages):
+    """Duplicate free list of GoNormalised Packages."""
+
+    def __init__(self, packages: List[Package], ecosystem: Ecosystem):
+        """Create NormalizedPackages by removing all duplicates from packages."""
+        self._packages = packages
+        self._ecosystem = ecosystem
+        self._dependency_graph: Dict[Package, Set[Package]] = defaultdict(set)
+        self._modules = []
+        self._version_map = {}
+        self.gh = GithubUtils()
```

Comment:
`gh` could be a local variable than an instance variable. It is not used anywhere except here in __init__
```suggestion
      gh = GithubUtils()
```

Reply:
Ack